moztelemetry/dataset.py

```diff
@@ -275,7 +306,12 @@ def records(self, sc, limit=None, sample=1, seed=42, decode=None, summaries=None
         total_size_mb = total_size / float(1 << 20)
         print("fetching %.5fMB in %s files..." % (total_size_mb, len(summaries)))
 
-        groups = _group_by_size_greedy(summaries, 10 * sc.defaultParallelism)
+        if group_by == 'equal_size':
+            groups = _group_by_equal_size(summaries, 10*sc.defaultParallelism)
+        elif group_by == 'greedy':
+            groups = _group_by_size_greedy(summaries, 10 * sc.defaultParallelism)
+        else:
+            groups = _group_by_size_greedy(summaries, 10*sc.defaultParallelism)
```

Comment:
Can you print something along the lines of "this group_by method is invalid, defaulting to greedy"? Alternatively, raising an exception on an invalid option. There isn't any feedback on whether it works in this case.

Reply:
To clarify, you would prefer an exception be raised in the `else` statement rather than a call to the greedy strategy. Am I correct with my understanding?